src/main/java/lotto/LottoNumber.java

```diff
@@ -0,0 +1,46 @@
+package lotto;
+
+import java.util.Objects;
+
+public class LottoNumber {
+    private static final int MIN_NUMBER = 1;
+    private static final int MAX_NUMBER = 45;
+
+    private final int value;
+
+    public LottoNumber(int value) {
+        validate(value);
+        this.value = value;
+    }
+
+    public LottoNumber(String value) {
+        this(Integer.parseInt(value));
+    }
```

Comment:
```suggestion
    public LottoNumber(String value) {
        this(Integer.parseInt(value));
    }
    
    public LottoNumber(int value) {
        validate(value);
        this.value = value;
    }
```

주 생성자를 마지막에 구현하는 것이 컨벤션